Dockerfile

```diff
@@ -1,3 +1,19 @@
+FROM alpine:latest AS build
+
+RUN apk add --no-cache github-cli
+
+ARG TARGETOS
+ARG TARGETARCH
+
+# https://github.com/distribution/distribution/actions/runs/5794478637
+ARG GH_RUN_ID=5794478637
+
+RUN --mount=type=secret,id=GITHUB_TOKEN \
+    GH_TOKEN=$(cat < /run/secrets/GITHUB_TOKEN) gh run download -R distribution/distribution "${GH_RUN_ID}" -n registry \
```

Comment:
I assume this downloads the artifact from the run? They expire after a certain period so this will definitely bite us fairly soon

Reply:
One thing we can do, aside from forking and potentially having to set up our own build pipelines, is to store/cache the build blob in git here @Page-